packages/core/src/layout/HeaderTabs/index.tsx

```diff
@@ -42,16 +42,17 @@ export type Tab = {
   id: string;
   label: string;
 };
+
 export const HeaderTabs: React.FC<{
   tabs: Tab[];
-  onChange?: (index: Number) => void;
+  onChange?: (index: number) => void;
 }> = ({ tabs, onChange }) => {
   const [selectedTab, setSelectedTab] = useState<Number>(0);
```

Comment:
This that this should be `useState<number>` and then you should be able to drop the `valueOf` from the number on L54

Reply:
Ah right, will do. I should have gone all the way to the source of the type instead of making a "shallow" fix.

Reply:
fix'd